pip install setuptools-69.0.3
WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x000001FAD1B3C5D0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /simple/setuptools-69-0-3/
WARNING: Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x000001FAD5300D10>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /simple/setuptools-69-0-3/
WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x000001FAD5303990>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /simple/setuptools-69-0-3/
WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x000001FAD5310250>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /simple/setuptools-69-0-3/
WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x000001FAD5310AD0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /simple/setuptools-69-0-3/
ERROR: Could not find a version that satisfies the requirement setuptools-69.0.3 (from versions: none)
ERROR: No matching distribution found for setuptools-69.0.3

(mqc_web) C:\Users\ruby\python\mqc_web\Lib\site-packages>pip install setuptools-69.0.3.tar.gz
Processing c:\users\ruby\python\mqc_web\lib\site-packages\setuptools-69.0.3.tar.gz
  Getting requirements to build wheel ... done
  Installing backend dependencies ... error
  ERROR: Command errored out with exit status 1:
   command: 'C:\Users\ruby\python\mqc_web\Scripts\python.exe' 'C:\Users\ruby\AppData\Local\Temp\5\pip-standalone-pip-dxw0vpsf\__env_pip__.zip\pip' install --ignore-installed --no-user --prefix 'C:\Users\ruby\AppData\Local\Temp\5\pip-build-env-8ulf38_4\normal' --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- wheel
       cwd: None
  Complete output (7 lines):
  WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x0000022F584B1B50>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /simple/wheel/
  WARNING: Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x0000022F584B2410>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /simple/wheel/
  WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x0000022F584B2E10>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /simple/wheel/
  WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x0000022F584B3890>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /simple/wheel/
  WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x0000022F584D0310>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /simple/wheel/
  ERROR: Could not find a version that satisfies the requirement wheel (from versions: none)
  ERROR: No matching distribution found for wheel
  ----------------------------------------
WARNING: Discarding file:///C:/Users/ruby/python/mqc_web/Lib/site-packages/setuptools-69.0.3.tar.gz. Command errored out with exit status 1: 'C:\Users\ruby\python\mqc_web\Scripts\python.exe' 'C:\Users\ruby\AppData\Local\Temp\5\pip-standalone-pip-dxw0vpsf\__env_pip__.zip\pip' install --ignore-installed --no-user --prefix 'C:\Users\ruby\AppData\Local\Temp\5\pip-build-env-8ulf38_4\normal' --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- wheel Check the logs for full command output.
ERROR: Command errored out with exit status 1: 'C:\Users\ruby\python\mqc_web\Scripts\python.exe' 'C:\Users\ruby\AppData\Local\Temp\5\pip-standalone-pip-dxw0vpsf\__env_pip__.zip\pip' install --ignore-installed --no-user --prefix 'C:\Users\ruby\AppData\Local\Temp\5\pip-build-env-8ulf38_4\normal' --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- wheel Check the logs for full command output.
